packages/stark-ui/src/modules/pretty-print/components/_pretty-print-theme.scss

```diff
@@ -0,0 +1,14 @@
+/* ============================================================================== */
```

Comment:
For the pretty-print component doesn't make sense to have a "theme" scss file. In fact these styles should be in a normal scss file, in this case `pretty-print.component.scss`.

You can ask @catlabs  to explain you the purpose of those 2 different files.

Reply:
@RobbyDeLaet Could you please remove this file since it is not useful ? 😊